code/src/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Header = (props) => {
```

Comment:
You do not use props inside of your function so it is no need to declare it here.